beacon-chain/rpc/eth/beacon/handlers.go

```diff
@@ -65,6 +65,11 @@ func (s *Server) GetBlockV2(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if err := blocks.BeaconBlockIsNil(blk); err != nil {
+		httputil.HandleError(w, fmt.Sprintf("block id %s was not found", blockId), http.StatusNotFound)
```

Comment:
Is this status ok? seems like previously this returned a status ok with an empty object